libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -1172,6 +1196,23 @@ bool AP_MotorsUGV::is_digital_pwm_type() const
     return false;
 }
 
+/*
+  handle delay on reversal for a throttle
+ */
+void AP_MotorsUGV::ReverseThrottle::output(SRV_Channel::Function function, float throttle, float delay)
+{
+    const uint32_t now_ms = AP_HAL::millis();
+    if (throttle * last_throttle < 0 &&
```

Comment:
What happens if either one of these is zero?  eg. could we NOT pause when w need to because last_throttle is zero here?

Reply:
fixed